src/ServiceV2/Mailer/MailerService.php

```diff
@@ -30,8 +34,10 @@ public function __construct(
         private readonly RouterInterface $router,
         private readonly LoggerInterface $logger,
         private readonly TranslatorInterface $translator,
+        private readonly Security $security,
         private readonly string $mailerSender,
         private readonly array $adminMails,
```

Comment:
We don't need to inject adminMails anymore

Reply:
oh nice !

Reply:
Well, we need it for personal data request emails.

Reply:
mybad